playbooks/roles/nginx/templates/edx/app/nginx/sites-available/edx_notes_api.j2

```diff
@@ -1,3 +1,5 @@
+{% if INSTALL_EDX_NOTES %}
```

Comment:
I don't understand this change. edx-notes will only be installed if `INSTALL_EDX_NOTES` is set to `True` and this template will only be included if `edx_notes_api` is in the `nginx_sites` list variable. 

Does this conditional serve any additional purpose?

Reply:
@tkeemon You're right it shouldn't be included. However by default [`INSTALL_EDX_NOTES` isn't defined so the role won't be installed](https://github.com/appsembler/configuration/blob/b80f1b9/playbooks/appsemblerPlaybooks/ficus_basic.yml#L51-L53).

However the [`edx_notes_api` is included by default in the nginx roles](https://github.com/appsembler/configuration/blob/b80f1b9/playbooks/appsemblerPlaybooks/ficus_basic.yml#L30-L35) all the time.

While sometimes this issue passes silently without explicit errors, the nginx does indeed make it into the server and creates errors (in specific scenarios).

Deleting the file manually might have done it for me in my very specific scenario, which I don't clearly remember, but the `edx_notes_api` shouldn't make it to the server anyway.